packages/contracts/contracts/L1/messaging/L1StandardBridge.sol

```diff
@@ -28,7 +29,7 @@ contract L1StandardBridge is IL1StandardBridge, CrossDomainEnabled {
      * External Contract References *
      ********************************/
 
-    address public override l2TokenBridge;
+    address public override immutable l2TokenBridge;
```

Comment:
As you alluded to elsewhere, this would change the storage layout. But we can make that work by adding an unused storage var.